hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/client/TokenClient.java

```diff
@@ -274,32 +276,56 @@ public NetworkTransactionResponse createNonFungibleToken(
     }
 
     public NetworkTransactionResponse associate(ExpandedAccountId accountId, TokenId token) {
-        TokenAssociateTransaction tokenAssociateTransaction = new TokenAssociateTransaction()
-                .setAccountId(accountId.getAccountId())
-                .setTokenIds((List.of(token)))
-                .setTransactionMemo(getMemo("Associate w token"));
-
-        var keyList = KeyList.of(accountId.getPrivateKey());
-        var response = executeTransactionAndRetrieveReceipt(tokenAssociateTransaction, keyList);
-        log.info("Associated account {} with token {} via {}", accountId, token, response.getTransactionId());
-        tokenAccounts.add(new TokenAccount(token, accountId));
-        return response;
+        var account = new EntityId(accountId);
+        TokenAccountPair key = new TokenAccountPair(token, account);
```

Comment:
I don't think we should have a second object that holds token plus account nor the EntityId object. Why can't you just do:
```suggestion
        var key = new TokenAccount(token, accountId);
```

Reply:
Great idea! Now it is updated